pkg/controller/upgradeconfig/cluster_upgrader.go

```diff
@@ -601,6 +606,9 @@ func performClusterHealthCheck(c client.Client) (bool, error) {
 
 	defer resp.Body.Close()
 	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return false, err
```

Comment:
As per earlier comment.
```suggestion
		return false, fmt.Errorf("Error when querying Prometheus alerts: %s", err)
```

Reply:
Thanks a lot for the review Matt ! Have submitted changes..